UnitsNet.Tests/CustomCode/MolarMassTests.cs

```diff
@@ -40,5 +41,17 @@ public class MolarMassTests : MolarMassTestsBase
         protected override double NanogramsPerMoleTolerance => 1e-3;
         protected override double NanogramsPerMoleInOneKilogramPerMole => 1e12;
         protected override double PoundsPerMoleInOneKilogramPerMole => 2.2046226218487757;
+
+        private const double MolarMassOfOxygen = 15.999;
```

Comment:
Suggestion of 15.9994 and moving/merging it to `KnownQuantities` still applies here.